DOCKER.md

```diff
@@ -1,7 +1,8 @@
 # Docker Tutorial
 1. Download and install docker.
 2. Clone the simstring-1.1 project. `git clone https://github.com/chokkan/simstring.git`
+3. Clone the Brat project. `git clone https://github.com/allenai/brat.git`
```

Comment:
simstring-1.1 should be cloned *inside* the brat clone, so I would swap steps 2 and 3 and add a "cd brat" inbetween.